assets/admin/tour/style.scss

```diff
@@ -11,3 +11,44 @@
 		height: 138px;
 	}
 }
+
+$shadow-width: 30px;
+$border-color: var(--wp--preset--color--foreground, #ff6ec7 );
```

Comment:
I suspect the color you want here would be more the `--wp--preset--color--primary` instead of the `--wp--preset--color--foreground`, right? I imagine in some cases the foreground will be black or gray and probably won't match the expectations to highlight the content as I imagine you wanted here.

Reply:
Good point, I've changed it here https://github.com/Automattic/sensei/pull/7535/commits/be0505bb15aeb0d8feca757bf502b8acd94ce541 . Thanks!